src/index.js

```diff
@@ -107,6 +116,7 @@ function validateXML(options) {
 			// #endif
 
 			const valid = validationSucceeded(data.exitCode);
+			worker.terminate();
```

Comment:
Added this as from local testing I was able to identify that the workers weren't getting cleaned up after completion.